src/NUnitFramework/framework/Assert.Equality.cs

```diff
@@ -80,10 +79,9 @@ public static void AreEqual(double expected, double? actual, double delta, strin
         /// <param name="expected">The expected value</param>
         /// <param name="actual">The actual value</param>
         /// <param name="delta">The maximum acceptable difference between the the expected and the actual</param>
+        [System.Obsolete("This method is going to be removed in NUnit 4.0")]
```

Comment:
Could you say `will be` and add a period to make it a full sentence? (Full sentence because that's what it would have to be if we add a second sentence, as IIRC we do elsewhere.)